events-common/src/main/java/net/opentsdb/events/EventWriter.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file is part of OpenTSDB.
+ * Copyright (C) 2021  Yahoo.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.opentsdb.events;
+
+import net.openhft.hashing.LongHashFunction;
+import net.opentsdb.events.view.Event;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.LinkedBlockingQueue;
+
+public abstract class EventWriter implements Indexer, Runnable {
+
+  private static final Logger LOG = LoggerFactory.getLogger(EventWriter.class);
+  public static DateTimeFormatter df = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static LinkedBlockingQueue<Event> queue = new LinkedBlockingQueue<>(10000);
```

Comment:
Why is it static?

Reply:
`10000` should be configurable?

Reply:
I don't think this is used. Same with the `run` in the class.